lib/prune-publish.js

```diff
@@ -0,0 +1,22 @@
+// SPDX-FileCopyrightText: 2022 Andre 'Staltz' Medeiros <contact@staltz.com>
+//
+// SPDX-License-Identifier: LGPL-3.0-only
+
+module.exports = function prunePublish(ssb, content, cb) {
+  const group = content.tangles.group
+
+  ssb.db.create({ content, encryptionFormat: 'box2' }, (err, msg) => {
+    if (err) {
```

Comment:
we could actually check the error message includes "Too large" or something, but what you've done is fine

Reply:
yeah but it'd have to work for all feed formats and then i'd have to find a spec somewhere and update it and eeh

Reply:
Create an issue for it, that would make us more comfortable merging this :)

Reply:
made one https://github.com/ssbc/ssb-tribes2/issues/25